census/ui_app/EntryForm.jsx

```diff
@@ -129,6 +129,9 @@ const EntryForm = React.createClass({
   },
 
   render() {
+    let readonly = (this.props.isReview && !this.props.canReview);
+    let readOnlyOpts = {};
+    if (readonly) readOnlyOpts.readOnly = 'readonly';
```

Comment:
I have seen various linters not liking this variant of `if` statement. If your linter is fine with it then okay, otherwise you can do `readOnlyOpts.readOnly = readonly ? 'readonly' : undefined;`. Though I'm not sure if an `else` value for this property help you at all.